astroid/scoped_nodes.py

```diff
@@ -1509,6 +1510,29 @@ def _valid_getattr(node):
                 pass
         return False
 
+    def getitem(self, index, context=None):
+        """Return the inference of a subscript.
+
+        This is basically looking up the method in the metaclass and calling it.
+        """
+        try:
+            methods = dunder_lookup.lookup(self, '__getitem__')
+        except exceptions.AttributeInferenceError as error:
+            util.reraise(exceptions.InferenceError(**vars(error)))
+
+        method = methods[0]
+        # Rewrap index to Const for this case
```

Comment:
This comment does not seem true in this context, since we are not wrapping anything in Const.